services/sidecar/src/simcore_service_sidecar/executor.py

```diff
@@ -140,11 +141,20 @@ async def _process_task_inputs(self):
                 for port in (await PORTS.inputs)
             ]
         )
-
-        if input_ports:
-            file_name = self.shared_folders.input_folder / "input.json"
-            file_name.write_text(json.dumps(input_ports))
         log.debug("Inputs parsing DONE")
+        return input_ports
+
+    async def _write_input_file(self, inputs: Dict) -> None:
+        if inputs:
+            log.debug("Writing input file...")
+            stem = (
```

Comment:
why lowercase here and upper case in the other case? I think both should be lowercase since it is not a const literal

Reply:
fixed.